Plugins/ThreadWatchdog/ThreadWatchdog.cpp

```diff
@@ -85,10 +94,19 @@ void ThreadWatchdog::MainLoopUpdate(Services::Hooks::CallType, API::CServerExoAp
                     // tasks, and so we remain productive until somebody can come along and figure out why we've
                     // hit a long stall.
                     tasks->ProcessWorkOnMainThread();
```

Comment:
Not related to your change, but this is actually pretty dangerous, and could turn a 15 second stall due to SQL into a crash pretty easily. There are major issues with not doing this (for one, the metric never goes through), so maybe we should consider raising the default interval to 60 seconds. If you've stalled for 60 seconds you may as well have crashed anyway.